lib/galaxy/tools/sort_collection_list.xml

```diff
@@ -1,8 +1,8 @@
 <tool id="__SORTLIST__"
-      name="Sort Collection"
+      name="Sort collection"
       version="1.0.0"
       tool_type="sort_collection">
-    <description>of list of datasets</description>
+    <description></description>
```

Comment:
```suggestion
    <description>alphabetically, numerically, or using predetermined order from a supplied file</description>
```

Reply:
I really do not want descriptions for collection tools. They all work on collections. If users want to learn more = read help section. It looks cleaner in the tool pane.